src/test/java/code/practice/tasks/FilesTaskTest.java

```diff
@@ -0,0 +1,94 @@
+package code.practice.tasks;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class FilesTaskTest {
+    private final String VALID_DATA_FILE_URI = "./src/test/resources/FilesTaskTest_1/ValidDataFile.txt";
```

Comment:
Please use `getClass().getClassLoader()..getResource(fileName)` to get file from `resource` folder.